src/main/config-store.js

```diff
@@ -24,6 +25,11 @@ export function createConfigStore() {
 					type: ['string', 'null'],
 					default: null,
 				},
+				coordinateFormat: {
+					type: ['string'],
+					enum: ['dd', 'dms', 'utm'],
+					default: 'utm',
```

Comment:
using the same default as mobile, although i vaguely remember some conversation about maybe using DD as the default instead.